lib/app/features/parent_side/child_details_page.dart

```diff
@@ -128,53 +128,52 @@ class _ChildDetailsPageState extends State<ChildDetailsPage>
           return [
             SliverAppBar(
               actions: [
-                if (model.image != null)
```

Comment:
Changes on this file mainly removing if statement on line 131, and reformatting the indentation of the code.